joblib/_memmapping_reducer.py

```diff
@@ -276,21 +324,23 @@ class ArrayMemmapReducer(object):
         same data array is passed to different worker processes.
     """
 
-    def __init__(self, max_nbytes, temp_folder, mmap_mode, verbose=0,
-                 prewarm=True):
+    def __init__(self, max_nbytes, temp_folder, mmap_mode,
+                 track_memmap_in_child, verbose=0, prewarm=True):
```

Comment:
`track_memmap_in_child` should be renamed to `unlink_memmap_on_gc_collect` as `ArrayMemmapReducer` has no notion of parent and child by itself. Only the callers do.

Reply:
I'll go with `unlink_memmap_on_gc_collect` for now. I like better `give_ownership_to_loader` though (a bit more declarative). WDYT?